Gemfile

```diff
@@ -10,6 +10,4 @@ gem 'html-proofer', '= 4.4.3'
 gem 'jekyll-seo-tag'
 
 gem 'rspec-core'
-gem 'rspec-expectations'
-
-gem "webrick", "~> 1.7"
```

Comment:
This does not do anything

Reply:
Confirming, this didn't affect anything for me locally.

Reply:
I have a hunch this was necessary at some point (having been looking at old blog posts of people trying to get jekyll working over the years) but i guess not anymore. Ty for your service, webrick.